nodejs/eks/cluster.ts

```diff
@@ -20,7 +20,7 @@ import which = require("which");
 
 import { VpcCni, VpcCniOptions } from "./cni";
 import { createDashboard } from "./dashboard";
-import { createNodeGroup, NodeGroup, NodeGroupBaseOptions, NodeGroupData } from "./nodegroup";
+import { createNodeGroup, InputTags, NodeGroup, NodeGroupBaseOptions, NodeGroupData } from "./nodegroup";
```

Comment:
Minor - but should this shared type be in a shared `utils.ts` or similar?

Reply:
Yup good point, I'll change this.